msteams-platform/messaging-extensions/build-api-based-message-extension.md

```diff
@@ -0,0 +1,462 @@
+                              description: The unique identifier of the repair
+                            title:
+                              type: string
+                              description: The short summary of the repair
+                            description:
+                              type: string
+                              description: The detailed description of the repair
+                            assignedTo:
+                              type: string
+                              description: The user who is responsible for the repair
+                            date:
+                              type: string
+                              format: date-time
+                              description: The date and time when the repair is scheduled or completed
+                            image:
+                              type: string
+                              format: uri
+                              description: The URL of the image of the item to be repaired or the repair process
+            post:
+              operationId: createRepair
+              summary: Create a new repair
+              description: Adds a new repair to the list with the given details and image URL
+              requestBody:
+                required: true
+                content:
+                  application/json:
+                    schema:
+                      type: object
+                      properties:
+                        title:
+                          type: string
+                          description: The short summary of the repair
+                        description:
+                          type: string
+                          description: The detailed description of the repair
+                        assignedTo:
+                          type: string
+                          description: The user who is responsible for the repair
+                        date:
+                          type: string
+                          format: date-time
+                          description: The optional date and time when the repair is scheduled or completed
+                        image:
+                          type: string
+                          format: uri
+                          description: The URL of the image of the item to be repaired or the repair process
+                      required:
+                        - title
+                        - description
+                        - assignedTo
+              responses:
+                '201':
+                  description: A successful response indicating that the repair was created
+   ```
+
+   </details>
+
+   If you've created an OpenAPI specification document, you need an Adaptive Card template for the app to respond to the get requests. If your building an App using Teams Toolkit or Developer portal for teams, the tools will extract the information from the OpenAPI specification document automatically.
```

Comment:
I think the AC requirement is worth highlighting--either as part 1b of this requirement, or maybe its own separate req

Reply:
Updated as part b.